### PR TITLE
[Merged by Bors] - [CU-2m9md6n] docs(book/ecosystem): edit team page 

### PR DESCRIPTION
**Pull request by Martin-Kiss**
## Issue
https://app.clickup.com/t/2m9md6n

## Description
Update content of team page in the mdbook.

## Checklist

~~- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_~~
~~- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_~~

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2m9md6n Edit Team Page on Mdbook](https://app.clickup.com/t/2m9md6n)

File: book/src/ecosystem/the-composable-team.md [hunk 1]
```diff
@@ -2,32 +2,41 @@
 
 ---
 
-The Composable Finance team is made up of individuals dedicated to realizing a seamless cross-chain user experience and a blockchain-agnostic future of DeFi. There are around 65 people on the Composable team, with 40 developers (including 15 on frontend or integration) and 25 on design, strategy and marketing. Select biographies on our core members are below. 
+
+
+The Composable Finance team is made up of individuals dedicated to realize seamless cross-chain user experience and a blockchain-agnostic future of DeFi. There are around 80 people on the Composable team; 40 developers, 12 on Product and QA, 8 on Growth and HR, and 18 on design, marketing and community. Select biographies on our core members are below.
```
Comment (benluelo) [inline book/src/ecosystem/the-composable-team.md:7]:
```suggestion
The Composable Finance team is made up of individuals dedicated to realizing seamless cross-chain user experience and a blockchain-agnostic future of DeFi. There are around 80 people on the Composable team; 40 developers, 12 on Product and QA, 8 on Growth and HR, and 18 on design, marketing and community. Select biographies on our core members are below.
```

File: book/src/ecosystem/the-composable-team.md [hunk 2]
```diff
@@ -39,52 +48,28 @@ Cosmin entered the crypto space in 2017, and has had a long career in computer s
 
 ## Seun Lanlege | Bridging Lead 
 
-Seun is bringing his wealth of experience as a former core developer at Parity Technologies to Picasso. Seun is leading the Polkadot team and harnessing his knowledge and experience to the growth of the Picasso Parachain. He is focused on the Picasso and Composable runtime as well as developing trustless bridging solutions between various networks, including DotSama, Cosmos, and more. Follow him on [Twitter](https://twitter.com/seunlanlege) or [Medium](https://medium.com/@seunlanlege).
+Seun is bringing his wealth of experience as a former core developer at Parity Technologies to Picasso. Seun is leading the bridging team and harnessing his knowledge and experience to the growth of developing trustless bridging solutions between various networks beyond DotSama including DotSama, IBC, and more. Follow him on [Twitter](https://twitter.com/seunlanlege) or [Medium](https://medium.com/@seunlanlege).
```
Comment (benluelo) [inline book/src/ecosystem/the-composable-team.md:51]:
```suggestion
Seun brings his wealth of experience as a former core developer at Parity Technologies to Picasso. Seun is leading the bridging team and harnessing his knowledge and experience to power the development of trustless bridging solutions between various networks beyond DotSama including DotSama, IBC, and more. Follow him on [Twitter](https://twitter.com/seunlanlege) or [Medium](https://medium.com/@seunlanlege).
```

Comment (benluelo) [reply book/src/ecosystem/the-composable-team.md:51]: "beyond DotSama including DotSama"?

**Review by benluelo — DISMISSED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]
Plz readd blas

**Review by benluelo — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Pull request closed without merging**